pkg/kubelet/rkt/rkt.go

```diff
@@ -2054,13 +2060,10 @@ func (r *Runtime) GetPodStatus(uid types.UID, name, namespace string) (*kubecont
 		}
 	}
 
-	if latestPod != nil {
-		// Try to fill the IP info.
-		for _, n := range latestPod.Networks {
-			if n.Name == defaultNetworkName {
-				podStatus.IP = n.Ipv4
-			}
-		}
+	// TODO(euank): this will not work in host networking mode
+	containerID := kubecontainer.ContainerID{ID: string(uid)}
```

Comment:
Just to be clean, so in the host network, the pod ip is not set here, but will be set in https://github.com/kubernetes/kubernetes/pull/24633/files , right?